srcpkgs/pam_rundir/patches/fix2.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/pam_rundir.c b/pam_rundir.c
```

Comment:
i assume you got this from here? https://github.com/jjk-jacky/pam_rundir/pull/5/
please put a reference, (same for the other patches)